### DISCUSSION TITLE
Ability to purchase multiple Supporter Tiers at the same time

### DISCUSSION THREAD

**Discussion by ssiwm** [gameserverapp/Platform#503] [Suggestions]
Useful when selling tokens, you will not have to create many with different values. The player will be able to buy a multiple of the same. Let's assume a multiple of 1000 Tokens instead of buying a package for 2000 and another for 5000.

**Comment by mvaessen** [comment]
hi, could you please move your suggestion for folders to a separate suggestion for me please?

**Reply by ssiwm** [reply]
sure, 

Folders to organize Supporter Tiers #507
